30412/Itu Anca/Assignment1/main.c

```diff
@@ -0,0 +1,76 @@
+#include "head.h"
+
+int main() {
+    fptrout = fopen("D:\\Facultate\\DSA\\Assignment1\\output.dat", "w");
+    if (fptrout == NULL)
+    {
+        printf("Could not open file");
+        return 0;
+    }
+
+    first = NULL;
+    last = NULL;
+
+    char c[100];
+    int n;
+
+    FILE * fptr;
+    fptr = fopen("D:\\Facultate\\DSA\\Assignment1\\input.dat", "r");
+    if(fptr == NULL){
+        printf("ERROR");
+        exit(1);
+    }
+
+    for(int i=1;i<=21;i++)
+    {
+        fscanf(fptr,"%s", c);
+        if(strcmp("AL",c)==0)
+        {
+            fscanf(fptr, "%d", &n);
+            AddLast(n);
+        }
+        if(strcmp("AF",c)==0)
```

Comment:
use `else` here, it doesn't make sense to check all other possibilities because it can only have one value at a time